README.md

```diff
@@ -1,318 +1,336 @@
+              minimumBet:
+                $ref: '#/definitions/minimumBet'
+              maximumBet:
+                $ref: '#/definitions/maximumBet'
+            required:
+              - playerNum
+              - minimumBet
+              - maximumBet
+            additionalProperties: false
+        required: 
+          - type
+          - attributes
+        additionalProperties: false
+    required:
+      - data
+    additionalProperties: false
+  gameType:
+    type: string
+    enum:
+      - game
+  playerType:
+    type: string
+    enum:
+      - player
+  playerID:
+    type: string
+    example: 'g1p1'
+    pattern: '^g[1-9]\d*p[1-9]\d*$'
+  gameId:
+    type: integer
+    minimum: 0
+    description: A unique ID of a poker game
+  playerNum:
+    type: integer
+    maximum: 5
+    minimum: 2
+    description: The number of player in the game
+    example: 2
+  minimumBet:
+    type: integer
+    minimum: 0
+    description: The minimum bet every player have to place at least each round.
+    example: 1000
+  maximumBet:
+    type: integer
+    minimum: 0
+    description: The maximum bet every player can place each round.
+    example: 2000
+  playerLinks:
+    type: array
+    items:
+      $ref: '#/definitions/playerLink'
+  playerLink:
+    type: string
+    format: url
+    description: Link to the playerResource
+  cards:
+    type: array
+    items:
+      $ref: '#/definitions/card'
+  cardLink:
+    type: string
+    format: url
+    description: Link to the cardResource
+  GameResult:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        $ref: '#/definitions/GameResource'
+  PlayerResult:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        $ref: '#/definitions/playerResource'
+  round:
+    type: string
+    enum:
+      - 'blind'
+      - 'flop'
+      - 'turn'
+      - 'river'
+      - 'showdown'
+  SelfLink:
+    properties:
+      self:
+        type: string
+        format: url
+        description: Self-link of current resource
+  
+  Error:
+    properties:
+      status:
+        type: string
+        description: HTTP status code
+        example: '123'
+      title:
+        type: string
+        description: A short, user readable summary of the error
+        example: Not Found
+      code:
+        type: string
+        description: An application-specific error code
+        example: '1234'
+      detail:
+        type: string
+        description: A long description of the error that may contain instance-specific details
+      links:
+        properties:
+          about:
+            type: string
+            format: url
+            description: A link to further information about the error
+            example: https://developer.oregonstate.edu/documentation/error-reference#1234
+  Errors:
+    properties:
+      errors:
+        type: array
+        items:
+          $ref: '#/definitions/Error'
+              
```

Comment:
I think you pasted your openapi design to the readme accidentally, can you fix it?